caos.ansible_roles/roles/nr_repo_setup/tasks/main.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 
 - ansible.builtin.include_tasks: "repo-{{ ansible_distribution }}.yaml"
-  when: ansible_distribution != "MacOSX"
-
-- name:
-  debug:
-    msg: "repo-MacOSX.yaml is not refactored to be generic yet"
-  when: ansible_distribution == "MacOSX"
+#   when: ansible_distribution != "MacOSX"
```

Comment:
As this is being fixed I would remove the comment :)

Reply:
Good catch, removed in https://github.com/newrelic-experimental/caos-ansible-roles/pull/68/commits/bcc8d782da8e841d3b65fdfc0fa464e08fe7c5e4